src/frontends/onnx/frontend/src/op/bitwise_and.cpp

```diff
@@ -10,20 +10,24 @@ OPENVINO_SUPPRESS_DEPRECATED_START
 using namespace ov::op;
 
 namespace ngraph {
-namespace onnx_import {
-namespace op {
-namespace set_1 {
-OutputVector bitwise_and(const Node& node) {
-    const auto inputs = node.get_ng_inputs();
-    OPENVINO_ASSERT(inputs.size() == 2);   
-    return {std::make_shared<v13::BitwiseAnd>(inputs[0], inputs[1])};
+	namespace onnx_import {
+		namespace op {
+			namespace set_1 {
+				OutputVector bitwise_and(const Node& node) {
+					const Output<ngraph::Node> a = node.get_ng_inputs().at(0);
+					const Output<ngraph::Node> b = node.get_ng_inputs().at(1);
 
-}
+					NGRAPH_CHECK(a.get_element_type() != ov::element::bf16 && b.get_element_type() != ov::element::bf16,
+						"The input data bfloat16 isn't supported in opset 12");
```

Comment:
Type validation is done inside `v13::BitwiseAnd` op constructor, so such check during conversion is redundant. Could you please back to the version from the previous commit? It was correct, just code style alignment needed:)

Reply:
yup my bad pushed the wrong commit